.github/workflows/ubuntu18.yml

```diff
@@ -0,0 +1,37 @@
+# This is a basic workflow to help you get started with Actions
+
+name: "Test Ubuntu 18 Support"
+
+# Controls when the action will run. 
+on:
+  # Triggers the workflow on push or pull request events but only for the v2.3_dev branch
+  push:
+    branches: [ main, indev, dev ]
+  pull_request:
+    branches: [ main, indev, dev ]
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  build:
+
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v4
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Update and Upgrade
+      run: |
+        sudo apt update && sudo apt upgrade -qqy
+        sudo apt install libcurl4-openssl-dev libssl-dev
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install flake8 pytest
+        if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
```

Comment:
Run setup for Ubuntu 18, run a matrix checking 4 python versions for pip dependencies and fail check if any deps can't be found/installed.

This help quickly check if a new updated package can't be found on specific python version on Ubuntu 18.